pints/_mcmc/_nuts.py

```diff
@@ -742,7 +737,11 @@ def tell(self, reply):
             self._next = next(self._nuts)
 
             # Return current position as next sample in the chain
-            return self._current
+            return (
+                np.copy(self._current),
+                (current_logpdf, np.copy(current_gradient)),
+                True
```

Comment:
I forgot how NUTS works, does it always accept?

Reply:
No, but this is the bit in the code where it accepts :D